Add ACR webhook receiver (#153)
fluxcd/notification-controller: docs/spec/v1beta1/receiver.md

    - kind: ImageRepository
      name: webapp
```

> Note that we do not verify the authenticity of the request as Azure doesn't provide any mechanism for verification. 

Reviewer: ```suggestion
Note that the controller doesn't verify the authenticity of the request as Azure doesn't provide any mechanism for verification. 
```
